src/style.css

```
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: Averin-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Averin-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+.input {
+  width: 100%;
+}
+
+.big {
+  font-family: Averin-book, serif;
+  font-size: 16px;
+  display: block;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  margin: 20px 0;
+  height: 70px;
+  background: #fff url("images/Search.svg") no-repeat scroll 26px 25px;
+  padding-left: 62px;
+  background-size: 19px;
+}
+
+.small {
```

```suggestion
.input--small {
```
2 point in checklist